bin/release

```diff
@@ -1,3 +1,35 @@
 #!/usr/bin/env bash
 # Usage: bin/release <build-dir>
 # See: https://devcenter.heroku.com/articles/buildpack-api
+
+set -euo pipefail
+
+BUILD_DIR="${1}"
+CNB_BUILD_DIR="${BUILD_DIR}/.heroku/cnb/dotnet"
+launch_toml_file="${CNB_BUILD_DIR}/layers/launch.toml"
+
+# Check if launch.toml exists and output default_process_types in YAML
+if [[ -f "${launch_toml_file}" ]]; then
+	echo "---"
+	echo "default_process_types:"
+
+	type=""
+	command=""
+
+	while IFS= read -r line; do
+		# Extract the type from lines like `type = "web"`
+		if [[ ${line} =~ ^type[[:space:]]*=[[:space:]]*\"([^\"]+)\" ]]; then
+			type="${BASH_REMATCH[1]}"
+		fi
+
+		# Extract the command from lines like `command = ["bash", "-c", "run_command"]`
+		if [[ ${line} =~ ^command[[:space:]]*=[[:space:]]*\[\"bash\",[[:space:]]*\"-c\",[[:space:]]*\"(.*)\"[[:space:]]*\] ]]; then
```

Comment:
Will serde/libcnb always serialise the command as one line, or can it ever be wrapped across multiple lines (eg if the command is long)?

Plus in general this is very coupled to the output format of serde/libcnb (for example if the `type` and `command` were ever reordered in the struct and the command written before the type in the resultant file, this implementation would break).

This is fine for now (and probably fine longer term, given that without having to vendor eq `yq` or similar there aren't many better options) - but mentioning for completeness.

Reply:
I've never seen serde/libcnb serialize commands differently (regardless of length), but reordered `type`/`command` (and multi-line) entries are definitely not supported in this implementation.

I'm fairly confident `launch.toml` entries won't be serialized in surprising ways, but agree this should be revisited. `yq` would make this a breeze for sure, but might make sense to explore ways the CNB can be extended to output `yaml`-formatted launch processes.

Reply:
In which case, this seems safe for now, given that the CNB version is pinned (so the classic buildpack won't pick up a struct reordering or serde version change without you bumping the CNB version manually).